mne/report.py

```diff
@@ -1097,10 +1095,11 @@ def add_figs_to_section(self, figs, captions, section='custom',
 
         Parameters
         ----------
-        figs : matplotlib.figure.Figure | mlab.Figure | array | list
+        figs : Figure | mlab.Figure | pyvista.Figure | array | list
             A figure or a list of figures to add to the report. Each figure in
             the list can be an instance of :class:`matplotlib.figure.Figure`,
-            :class:`mayavi.core.api.Scene`, or :class:`numpy.ndarray`.
+            :class:`mayavi.core.api.Scene`, ``_Figure``, or
```

Comment:
but what is `_Figure` ? isn't it a private class that the user should not be aware of?

Reply:
probably need to update the error message too

Reply:
`_Figure`  is the pyvista backend figure handle. I don't think it's ready to be public yet, not until https://github.com/mne-tools/mne-python/pull/8730#issuecomment-758077244 is solved

Reply:
That's why Circle complains actually

Reply:
I see, but doesn't `plot_alignment` also return a `Pyvista` figure handle?

Reply:
`plot_alignment` also returns an instance of the private class `_Figure`, `PyVista` does not have such a thing.